swagger.yaml

```diff
@@ -1072,6 +1084,12 @@ paths:
           type: string
           format: uuid
           required: true
+        - in: header
+          name: x-host-id
```

Comment:
why do we need both ``host_id`` in path and ``x-host-id`` in the header?

Reply:
We don't strictly need both, but it is convenient to have the header in all requests because it simplifies and makes more uniform the Envoy and Limitador configurations.